tests/issues/issue-317/issue-317.test.ts

```diff
@@ -0,0 +1,94 @@
+import { readFileSync } from 'fs';
+import { resolve } from 'path';
+import { format } from 'prettier';
+import { plugin } from './../../../src/index';
+
+describe('Issues', () => {
+	test('should respect printWidth and pugClassLocation', () => {
+		const expected: string = readFileSync(resolve(__dirname, 'formatted.pug'), 'utf8');
+		const code: string = readFileSync(resolve(__dirname, 'unformatted.pug'), 'utf8');
+		const actual: string = format(code, {
+			parser: 'pug',
+			plugins: [plugin],
+
+			arrowParens: 'always',
+			bracketSpacing: true,
+			printWidth: 120,
+			semi: true,
+			singleQuote: true,
+			trailingComma: 'none',
+			tabWidth: 2,
+			useTabs: false,
+
+			pugSingleQuote: false,
+			pugFramework: 'vue',
+			pugSortAttributesBeginning: [
```

Comment:
I think you can remove this huge block. I mostly try to shrink the config for the tests down to the real problem, so only the necessary part is left over. Then you can also remove the "known-words" from eslint again, due to these are really only project specific :thinking: 
We could think about to run spellcheck only on `src`, but on the other side it's nice to have everywhere.